plugins/out_s3/s3.c

```diff
@@ -1651,6 +1823,16 @@ static struct flb_config_map config_map[] = {
      "Send the Content-MD5 header with object uploads, as is required when Object Lock is enabled"
     },
 
+    {
+     FLB_CONFIG_MAP_STR, "log_key", NULL,
+     0, FLB_FALSE, 0,
+     "By default, the whole log record will be sent to S3. "
+     "If you specify a key name with this option, then only the value of "
+     "that key will be sent to S3. For example, if you are using "
+     "the Fluentd Docker log driver, you can specify log_key log and only "
```

Comment:
Nit: "For example, if you are using Docker, you can specify `log_key log` and only"

Fluentd docker log driver is too specific, and its not the only thing which adds this log key